zetaclient/tss_signer.go

```diff
@@ -273,14 +273,16 @@ func (tss *TSS) SignBatch(digests [][]byte, height uint64, nonce uint64, chain *
 
 	if ksRes.Status == thorcommon.Fail {
 		log.Warn().Msg("keysign status FAIL posting blame to core")
-		digest := combineDigests(digestBase64)
-		index := observertypes.GetBlameIndex(chain.ChainId, nonce, hex.EncodeToString(digest), height)
+		//digest := combineDigests(digestBase64)
+		//index := observertypes.GetBlameIndex(chain.ChainId, nonce, hex.EncodeToString(digest), height)
 
-		zetaHash, err := tss.CoreBridge.PostBlameData(&ksRes.Blame, chain.ChainId, index)
-		if err != nil {
-			log.Error().Err(err).Msg("error sending blame data to core")
-			return [][65]byte{}, err
-		}
+		//zetaHash, err := tss.CoreBridge.PostBlameData(&ksRes.Blame, chain.ChainId, index)
+		//if err != nil {
+		//	log.Error().Err(err).Msg("error sending blame data to core")
+		//	return [][65]byte{}, err
+		//}
+		//
+		//log.Info().Msgf("keysign posted blame data tx hash: %s", zetaHash)
```

Comment:
Can we create an issue to put this back or remove the comment completely if no longer relevant?

Reply:
Better to have a flag to enable/disable